Khan/perseus
# [Locked Figure Labels] Add color select to locked figure labels settings

## Summary:
Add back the color select into the locked figure labels. The content authors
would be able to use the color select to specify colors separately from the
figure color itself.

Charlie made a good point:
> I like having the dropdown option. I imagine it would primarily be useful in the case where a label overlaps another figure, where we might need to choose a color based on having adequate color contrast with that other figure. (I hope that case would be infrequent, since it would also introduce labeling ambiguities for sighted users.)

Issue: https://khanacademy.atlassian.net/browse/LEMS-2431

## Test plan:
`yarn jest`

Storybook
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-figure-labels-all-flags
- Check that all the locked labels have color selects
- Confirm that the color select changes the color
- Confirm that changing the overall figure's color select still overrides the label color


<img width="389" alt="Screenshot 2024-09-27 at 3 35 05 PM" src="https://github.com/user-attachments/assets/84c7ec2d-2e52-4e3d-9299-1045cd220698">
